Source/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>5.0.0</string>
+	<string>5.0.0-beta.1</string>
```

Comment:
This didn't match our podspec in `master`. Was that intentional? I updated it here to match assuming that it wasn't.

Reply:
I thought that beta version strings would break the ability to ship apps with the framework. If that's not the case, the change is fine.